packages/aws-library/src/aws_library/ec2/_client.py

```diff
@@ -68,10 +68,10 @@ async def get_ec2_instance_capabilities(
         self,
         instance_type_names: set[InstanceTypeType],
     ) -> list[EC2InstanceType]:
-        """returns the ec2 instance types from a list of instance type names
-            NOTE: the order might differ!
+        """Returns the ec2 instance types from a list of instance type names (sorted by name)
+
         Arguments:
-            instance_type_names -- the types to filter with
+            instance_type_names -- the types to filter with. If an empty set, it returns all.
```

Comment:
```suggestion
            instance_type_names -- the types to filter with. If empty, it returns all.
```

Reply:
so I guess you ran all tests in autoscaling and clusters-keeper to ensure this does not change any behavior?